pages/contact-us/contact-employment-insurance.js

```diff
@@ -0,0 +1,249 @@
+import { getProfileContent } from '../../graphql/mappers/profile'
+import { getBetaBannerContent } from '../../graphql/mappers/beta-banner-opt-out'
+import { getBetaPopupExitContent } from '../../graphql/mappers/beta-popup-exit'
+import { getBetaPopupNotAvailableContent } from '../../graphql/mappers/beta-popup-page-not-available'
+import logger from '../../lib/logger'
+import Modal from 'react-modal'
+import React from 'react'
+import ExitBetaModal from '../../components/ExitBetaModal'
+import ContactProvince from '../../components/ContactProvince'
+import Markdown from 'markdown-to-jsx'
+const PageData = require('../../json/Nov30Data.json')
+
+const tmpContactMethods = PageData.data.schPagev1ByPath.item
+
+const pareseContactMethods = (rawMethods) => {
+  let tmp = {
+    en: {
+      breadCrumbs: rawMethods.scBreadcrumbParentPages.map((x) => {
+        return {
+          text: x.scTitleEn,
+          link: `/${x.scPageNameEn}`,
+        }
+      }),
+      title: rawMethods.scTitleEn,
+      id: rawMethods.scId,
+      methods: rawMethods.scFragments[0].scItems.map((x) => {
+        return {
+          title: x.scTitleEn,
+          intro: x.schIntroEn.markdown,
+          id: x.scId,
+          details: x.schDetails.map((x) => {
+            return {
+              label: x.scTitleEn,
+              id: x.scId,
+              detail: x.scItems[0] ? x.scItems[0].scContentEn.markdown : null,
+            }
+          }),
+        }
+      }),
+    },
+    fr: {
+      breadCrumbs: rawMethods.scBreadcrumbParentPages.map((x) => {
+        return {
+          text: x.scTitleEn,
+          link: `/${x.scPageNameEn}`,
+        }
+      }),
+      title: rawMethods.scTitleFr,
+      id: rawMethods.scId,
+      methods: rawMethods.scFragments[0].scItems.map((x) => {
+        return {
+          title: x.scTitleFr,
+          intro: x.schIntroFr.markdown,
+          id: x.scId,
+          details: x.schDetails.map((x) => {
+            return {
+              label: x.scTitleFr,
+              id: x.scId,
+              detail: x.scItems[0] ? x.scItems[0].scContentFr.markdown : null,
+            }
+          }),
+        }
+      }),
+    },
+  }
+  tmp.en.mail = tmp.en.methods[3]
+  tmp.en.mail.details = tmp.en.mail.details.map((x) => {
+    return {
+      province: x.label,
+      id: x.detail,
+      contentEi:
+        'Service Canada\n\nEmployment Insurance Program\n\nPO Box 2100\n\nVanvouver BC V6B 3T4',
+      contentDocuments:
+        'Service Canada Center\n\nPO Box 245\nEdmonton AB T5J 2J1',
+    }
+  })
+  tmp.fr.mail = tmp.fr.methods[3]
+  tmp.fr.mail.details = tmp.fr.mail.details.map((x) => {
+    return {
+      province: x.label,
+      id: x.detail,
+      contentEi:
+        'Service Canada\n\nEmployment Insurance Program\n\nPO Box 2100\n\nVanvouver BC V6B 3T4',
+      contentDocuments:
+        'Service Canada Center\n\nPO Box 245\nEdmonton AB T5J 2J1',
+    }
+  })
+  tmp.en.methods.pop()
+  tmp.fr.methods.pop()
+  return tmp
+}
```

Comment:
Take a peek on how AEM mappers separate out this logic from the page

Reply:
This will be addressed when we connect the GraphQL query